docs/reference/aggregations/metrics/scripted-metric-aggregation.asciidoc

```diff
@@ -225,13 +225,4 @@ params::           Optional. An object whose contents will be passed as variable
 --------------------------------------------------
 reduce_params::    Optional. An object whose contents will be passed as variables to the `reduce_script`. This can be useful to allow the user to control 
                    the behavior of the reduce phase. If this is not specified the variable will be undefined in the reduce_script execution.
-lang::             Optional. The script language used for the scripts. If this is not specified the default scripting language is used.
-init_script_file:: Optional. Can be used in place of the `init_script` parameter to provide the script using in a file.
-init_script_id:: Optional. Can be used in place of the `init_script` parameter to provide the script using an indexed script.
-map_script_file:: Optional. Can be used in place of the `map_script` parameter to provide the script using in a file.
-map_script_id:: Optional. Can be used in place of the `map_script` parameter to provide the script using an indexed script.
-combine_script_file:: Optional. Can be used in place of the `combine_script` parameter to provide the script using in a file.
-combine_script_id:: Optional. Can be used in place of the `combine_script` parameter to provide the script using an indexed script.
-reduce_script_file:: Optional. Can be used in place of the `reduce_script` parameter to provide the script using in a file.
-reduce_script_id:: Optional. Can be used in place of the `reduce_script` parameter to provide the script using an indexed script.
 
```

Comment:
This doc is left without any examples of non-inline scripts. I presume "id" and "file" options are possible but the doc doesn't link to examples of these?
I presume any `params` are global so passed to all init/map/combine/reduce - can their be type-specific params nested under each *_script variant?